dashboard/cron.py

```diff
@@ -280,6 +280,7 @@ def update_with_bq_access(self):
             job_config = bigquery.QueryJobConfig()
             job_config.query_parameters = query_params
 
+            logger.debug(final_bq_query)
```

Comment:
We have this on line 269, and I don't think the value changes after that point, correct?